include/wx/gtk/textctrl.h

```diff
@@ -133,6 +133,8 @@ class WXDLLIMPEXP_CORE wxTextCtrl: public wxTextCtrlBase
     bool SetForegroundColour(const wxColour& colour) override;
     bool SetBackgroundColour(const wxColour& colour) override;
 
+    void HandleMaxLength(wxString &text);
```

Comment:
Should have `GTK` prefix as it's GTK-specific and a comment explaining what it does would be welcome.

It should probably also return `bool` to indicate whether it changed the text to avoid useless calls to `SetValue()`.